INSTALL.md

```diff
@@ -188,7 +197,7 @@ CMake directly:
     cd build
     cmake ..
 
-If the compile stops because of warnings, try again with 
+If the compile stops because of warnings, try again with
 
     make nuke
     make STOP_ON_WARNING=0
```

Comment:
This is more a note to myself than you, but:  I have gotten in the habit of setting `STOP_ON_WARNING=0` in release branches by default. So it's only when working from master that a compiler warning might break the build.